### PR TITLE
adding todo-app example

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: example-apps/kitchen-sink-example/source/components/todo-app/todo-app.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import {Component, View, DirectiveResolver} from 'angular2/core';
+
+import {TodoService} from './todo-service';
+import {TodoInput} from './todo-input';
+import {TodoList} from './todo-list';
+
+const directiveResolver: DirectiveResolver = new DirectiveResolver();
+console.log(directiveResolver.resolve(TodoInput));
```
Comment (vanessayuenn) [inline example-apps/kitchen-sink-example/source/components/todo-app/todo-app.ts]: remove

File: example-apps/kitchen-sink-example/source/components/todo-app/todo-input.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+})
+export class TodoInput {
+  todoModel: TodoModel = new TodoModel();
+
+  constructor(
+    public todoService: TodoService,
+    public formatService: FormatService
+  ) {
+    console.log(todoService);
+    console.log(formatService);
```
Comment (vanessayuenn) [inline example-apps/kitchen-sink-example/source/components/todo-app/todo-input.ts]: remove

File: example-apps/kitchen-sink-example/source/components/todo-app/todo-list.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    </p>
+  </div>
+  `
+})
+export class TodoList {
+
+  constructor(
+    public todoService: TodoService
+  ) {
+    console.log(todoService);
```
Comment (vanessayuenn) [inline example-apps/kitchen-sink-example/source/components/todo-app/todo-list.ts]: :put_litter_in_its_place:

File: example-apps/kitchen-sink-example/source/components/todo-app/todo-service.ts [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    new TodoModel('three')
+  ];
+
+  addTodo(value: any): void {
+    this.todos.push(value);
+  }
+}
+
+export class FormatService {
+  sayHello() {
```
Comment (vanessayuenn) [inline example-apps/kitchen-sink-example/source/components/todo-app/todo-service.ts]: do we need this?

Comment (sumitarora) [inline example-apps/kitchen-sink-example/source/components/todo-app/todo-service.ts]: this is just for checking service injection in Batarangle

**Comment by vanessayuenn** [comment]
:shipit:

**Pull request merged**